.github/workflows/ci.yml

```diff
@@ -63,6 +67,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          lfs: 'true'
```

Comment:
I don't think we need this for the snapshot publishing

Reply:
Added the changes here for lfs checkout: 992b80dd7906e3e8b7bf586d7afed8d3bf7cfe69